TripPlanner - Project/TripPlannerFrontend/src/components/Layout/Header.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { DefaultLayout } from './DefaultLayout';
+import { Typography } from '../Typography/Typography';
+import Logo from '../Branding/Logo';
+
+const testUser = {
+  isLogged: false,
+  name: 'Bill',
+  surname: 'Gates',
+};
+
+export default function Header() {
+  const [users, setUsers] = useState(testUser);
+
+  function changeLogin() {
+    if (users.isLogged === false) {
+      setUsers((prevState) => ({
+        ...prevState,
+        isLogged: true,
+      }));
+    } else {
+      setUsers((prevState) => ({
+        ...prevState,
+        isLogged: false,
+      }));
+    }
+  }
```

Comment:
Nice destructuring! Here's shorter version though 😄 

```suggestion
  const toggleIsUserLogged = () => {
    setUsers((prevState) => ({
      ...prevState,
      isLogged: !prevState.isLogged,
    }));
  }
```

Reply:
Dzięki, zmieniłem zgodnie z sugestią :)